tendermint/starport
# refactor(scaffold): refactor `cosmosver` and print the migration guide

close #1548

### What does this MR does?

Convert the migration guide to HTML to shows the user how to update any version mismatch.

### How to test?

- Try to scaffold anything into a chain with an SDK version older them `v0.43.0` with any scaffold error. You will see the migration message.

- If you scaffold something into an old chain without errors or into a new chain with errors, the message does not appear.
